<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>170</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QLabel" name="label_user">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pwd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="margin">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="editline_user">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="editline_pwd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_browser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>260</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_login">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_signup">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
